Framework/Parallel/CMakeLists.txt

```diff
@@ -67,7 +76,37 @@ set_property ( TARGET Parallel PROPERTY FOLDER "MantidFramework" )
 
 target_include_directories ( Parallel SYSTEM PRIVATE ${HDF5_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
 target_link_libraries ( Parallel LINK_PRIVATE ${TCMALLOC_LIBRARIES_LINKTIME}
-                        ${GSL_LIBRARIES} ${MANTIDLIBS} ${HDF5_LIBRARIES} )
+                        ${GSL_LIBRARIES} ${MANTIDLIBS} ${HDF5_LIBRARIES} Kernel)
```

Comment:
It's because I use one of the kernel singletons to determine the binary directory of MantidPlot to find the binary for child process I need for multiprocessing.